### PR TITLE
add mobile-views

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam take a look

### CODE REVIEW

**Comment by xavijam** [comment]
I'd to:
- create a helper to know if an element has to be hidden on mobile/tablet/whatever.
- not repeat same content for the dashboard-view, maybe a new header should be added for mobile view (the top one).

File: themes/scss/widgets/canvas.scss [hunk 1]
```diff
@@ -85,13 +96,25 @@
...
     display: none;
   }
+  .CDB-Widget-canvasInner {
+    @include display-flex();
+    position: relative;
+  }
 }
 
 @include media-query-mobile() {
+  .Widget-canvas {
```
Comment (xavijam) [inline themes/scss/widgets/canvas.scss]: What about the ```CDB-``` prefix?

**Comment by viddo** [comment]
retest this please

**Comment by xavijam** [comment]
For me it is looking good

**Comment by xavijam** [comment]
:+1:

**Comment by viddo** [comment]
Failing tests… :sob:

**Comment by xavijam** [comment]
:joy:

**Pull request merged**